### DISCUSSION TITLE
Airflow webpage is giving 504 Gateway timed out error

### DISCUSSION THREAD

**Discussion by Rupam31592** [apache/airflow#29934] [General]
Airflow webpage is giving 504 Gateway timed out error. But I can see the logs in Kubernetes that jobs are getting scheduled and queued. Please help. Its happening in production so the impact is huge.

**Comment by potiuk** [comment]
This is a problem with you gateway in your deployment not airflow. Talk to your administratora

**Reply by Rupam31592** [reply]
@potiuk I got these logs inside webserver pod, I am bit doubtful that its a gateway issue.
![image](https://user-images.githubusercontent.com/26795880/223084870-7a159f4f-ff4f-4fd2-8164-e822ab9a13d0.png)

**Reply by potiuk** [reply]
Then you likely have problem with some resources (likely memory) that keeps on killing your PODs. But this is something that only monitoring of your deployment can tell (same story - contact your admins unless you are one, then look at your deployment issues).

**Reply by Rupam31592** [reply]
Hey @potiuk we have huge number of dags. can that be a problem? I have tried pointing to a different file share where the dag number is much less. then webserver got up and we could see the ui. but after pointing to the previous file share webserver crashed again. same worker timeout issue.

**Reply by potiuk** [reply]
Depends which version of Airflow you have. If you gave not kept your Airflow up-to-date, there were some  cases where there were some efficiency issues.

If you **think** your problem is related to airflow, not the gateway, then upgrading to latest version of airflow (2.5.1 currently) is the way to go.
